src/Symfony/Component/Process/Process.php

```diff
@@ -422,6 +429,8 @@ public function signal($signal)
     public function disableOutput()
     {
         if ($this->isRunning()) {
+            $this->stop(0);
```

Comment:
this drastically changes the bahavior of Symfony Process. I consider this as a BC break